learning_resources_search/tasks.py

```diff
@@ -768,7 +783,9 @@ def get_update_learning_resource_tasks(resource_type):
     ]
 
 
-@app.task(autoretry_for=(RetryError,), retry_backoff=True, rate_limit="600/m")
+@app.task(
+    acks_late=True, autoretry_for=(RetryError,), retry_backoff=True, rate_limit="600/m"
```

Comment:
I added acks_late=True here just in case, although I don't think this job failing is how orphaned indexes are generally created